deps

```diff
@@ -3,7 +3,7 @@
 ROOT=`pwd`
 
 BAZEL_DEPS_REPO="https://github.com/input-output-hk/bazel-deps.git"
-BAZEL_DEPS_COMMIT="b9baec69bdbaae53743a7c0b69828a9c66f6a624"
+BAZEL_DEPS_COMMIT="c9f440d823c987ce6ad86faf10fd53ecf6d9bbc7"
```

Comment:
Why do we need the change?

Reply:
source code scanning this commit is required